.eslintrc.js

```diff
@@ -1,5 +1,8 @@
 module.exports = {
   root: true,
+  parserOptions: {
+    ecmaVersion: 2018, // to support object rest spread, e.g. {...x, ...y}
```

Comment:
I copied this from eth-trezor-keyring in order to enable the spread operator. I don't think it should add any compatibility issues.